client/packages/common/src/ui/components/inputs/DatePickers/BaseDatePickerInput/BaseDatePickerInput.tsx

```diff
@@ -20,7 +20,7 @@ const TextField = (params: TextFieldProps) => {
     ...params,
     variant: 'standard',
   };
-  return <BasicTextInput {...textInputProps} />;
+  return <DeprecatedBasicTextInput {...textInputProps} />;
```

Comment:
Created a DeprecatedBasicTextInput for the Datepicker since there is a bug that currently doesn't handle slot and slotProps properly for datepicker input. There is a PR open for this, but it hasn't been updated in a month...